### PR TITLE
Oe 738/add hybrid shell

### PR DESCRIPTION
**Pull request by kristojorg**
This PR adds a basic capacitor starter app. I used the basic getting started setup from the capacitor documentation to set this us. That means it is using [vite](https://vitejs.dev) instead of Parcel.js to build the web code. This seems fine to me. Vite is actually ~10x more popular in download statistics than parcel, and appears to be more heavily maintained as well. It works very fast and leverages modern browser capabilities, which is especially good because we know this app will only ever run on modern browser tech. 

I named the app `oeh` because I like typing only 3 letters haha, but we can name it oe-hybrid or even open-ebooks-hybrid if y'all prefer. 

It should now be able to run in production mode on both ios and android by running `npm run build` then `npx cap sync` then `npx cap run ios` or `npx cap run android`. In dev mode with live reloading it currently only works on iOS. See readme for info about how to run that and let me know if it isn't clear : )

### CODE REVIEW

**Review by samanthaandrews — APPROVED** [review]

File: apps/oeh/README.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
+# Open eBooks Hybrid Mobile
+
+This app was created using [`@capacitor/create-app`](https://github.com/ionic-team/create-capacitor-app),
+and comes with a very minimal shell for building an app.
+
+## About
+
+- [Vite](https://vitejs.dev) to build the web code.
```
[the remark this answers is not in the document]
Comment (samanthaandrews) [reply apps/oeh/README.md:8]: ignore this comment i just read the PR description!

File: apps/oeh/android/app/src/androidTest/java/com/getcapacitor/myapp/ExampleInstrumentedTest.java [hunk 2]
```diff
@@ -0,0 +1,26 @@
+package com.getcapacitor.myapp;
```
[the remark this answers is not in the document]
Comment (kristojorg) [reply apps/oeh/android/app/src/androidTest/java/com/getcapacitor/myapp/ExampleInstrumentedTest.java:1]: It was added automatically when I ran the init method for a new capacitor app

File: apps/oeh/README.md [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+
+You can then run `npx cap run ios` or `npx cap run android` to build the native code and launch the simulator. Hybrid code changes will live reload and show automatically in the simulator.
+
+##### Note on HTTPS and Android
+
+Android strictly requires remote content to be loaded over HTTPS in a webview. This is true even in development and emulators. We can build the app into static files and run those on Android without an issue. However, when trying to run the development server, since the dev server is over http, it doesn't work.
+
+I have tried using Vite's `basicSSL` plugin to enable local HTTPS with a self-signed certificate, but it appears that Android does not accept this either. To try that method, uncomment the `basicSSL` and `https` lines in `vite.config.ts`, then change `server.url` to be `https://localhost:8100` in `capacitor.config.ts`.
+
+I believe there is a solution to this, likely a setting in the android project, but I don't know where it is. Help would be appreciated.
```
[the remark this answers is not in the document]
Comment (samanthaandrews) [reply apps/oeh/README.md:41]: https://jira.nypl.org/browse/OE-759

**Review by samanthaandrews — APPROVED** [review]
🚀

File: apps/oeh/android/app/src/androidTest/java/com/getcapacitor/myapp/ExampleInstrumentedTest.java (hunk 2 above)
[the remark this answers is not in the document]
Comment (samanthaandrews) [reply apps/oeh/android/app/src/androidTest/java/com/getcapacitor/myapp/ExampleInstrumentedTest.java:1]: ok good to know, thanks

**Pull request merged**